src/matter/common/FabricIndex.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BoundedUnsignedIntT, Typed } from "../../codec/TlvObjectCodec";
+
+/**
+ * Each fabric supported on a node is referenced by fabric-index that is unique on the node. This
+ * fab­ric-index enables the look-up of the full fabric information from the fabric-index. A fabric-index
+ * of 0 (zero) or null SHALL indicate that there is no fabric associated with the context in which the
+ * fab­ric-index is being used. If fabric-index is used in a context that is exclusively associated with
+ * a fab­ric, such as fabric-scoped data model elements, then the fabric-index values SHALL NOT include 0
+ * (zero) or null.
+ * 
+ * @see [Matter Specification R1.0], section 7.5.2
```

Comment:
"section" or § as above? We should decide for one ;-)

Reply:
Done!